src/routes/examples/advanced/multi-item-checkboxes/+page.svelte

```diff
@@ -7,12 +7,16 @@
         { value: 'three', label: 'Three' },
     ];
 
-    let value = [];
-    let checked = [];
-    let isChecked = {};
+    let value = $state([]);
+    let checked = $state([]);
+    let isChecked = $state({});
 
-    $: computeValue(checked);
-    $: computeIsChecked(checked);
+    $effect(() => {
+        computeValue(checked);
+    });
+    $effect(() => {
+        computeIsChecked(checked);
+    });
```

Comment:
Check if these can be changed to use $derived states without any warnings or errors, then you won't have $effect at all and no infinite loops 

Current:
```
 let value = $state([]);
    let checked = $state([]);
    let isChecked = $state({});
    $: computeValue(checked);
    $: computeIsChecked(checked);
    $effect(() => {
        computeValue(checked);
    });
    $effect(() => {
        computeIsChecked(checked);
    });
    function computeIsChecked() {
        isChecked = {};
        checked.forEach((c) => (isChecked[c] = true));
    }
    function computeValue() {
        value = checked.map((c) => items.find((i) => i.value === c));
    }
```

To
```

    let checked = $state([]);
    let value = $derived.by(()=> checked.map((c) => items.find((i) => i.value === c)));
    let isChecked = $derived.by(()=> {
        let r = {};
	checked.forEach((c) => (r[c] = true))
        return r;   
    });
```

Reply:
The cause of the infinite loop does not coming from a $effect that using on a +page.svelte because even I try comment out both of $effect in +page.svelte a issue of infinite loop still persist.

but dont be worry I will try tracking on this issue because it a last page that svelte-select should work 😁

Reply:
Well either way still it might help reduce lines of code and complexity, if what I proposed works as expected